databases/week1/databases_week1.sql

```diff
@@ -0,0 +1,64 @@
+-- Databases lesson 1 Homework
+USE hyf_class2023;
+
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(id) AS "TOTAL TASKS"
+FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT count(title) AS "Task Without Valid Due Date"
+FROM task
+WHERE due_date IS NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT title As "Tasks", name As "Status"
+FROM task
+INNER JOIN status
+ON task.status_id = status.id
+WHERE name = "Done";
+
+-- 4. Find all the tasks that are not marked as done
+SELECT title AS "Tasks", name AS "Status"
+FROM task 
+INNER JOIN status
+ON task.status_id = status.id
+WHERE NOT name = "Done";
+
+-- 5. Get all the tasks, sorted with the most recently created first
+SELECT title, created 
+FROM task
+ORDER BY created DESC;
+
+-- 6. Get the single most recently created task
+SELECT title, created 
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+-- 7. Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date
+FROM task
+WHERE (title like '%database%')
+OR
+(description like '%database%');
```

Comment:
In this case, the brackets give you nothing. 

check this link for a better understanding of why and when the brackets are important 
https://www.techonthenet.com/sql/and_or.php